ros/angel_msgs/msg/HeadsetPoseData.msg

```diff
@@ -0,0 +1,9 @@
+#
+# Message for transporting the world matrices and projection matrices provided
+# with each frame from the HoloLens.
+#
+
+std_msgs/Header header
+
+float32[] world_matrix
+float32[] projection_matrix
```

Comment:
Did you happen to consider the `PoseStamped` message from geometry_msgs? It does use quaternions though, so I'm not sure if that was the blocker that prevented it's use here (though I can find a reference for the quaternion math I needed to implement once in java to use this message type).

Reply:
Yeah, I looked at the `geometry_msgs/Pose` and `geometry_msgs/Transform` a little bit, but wasn't exactly sure how they related to the 4x4 matrices. It seems that the 4x4 matrices are composed of a translation vector and a 3x3 rotation matrix? I wasn't sure how to go from that to a quaternion. And then if we'd end up just converting the quaternion back to the world matrix to do all of the math.